Remove Xenial docs and update PHP 7.2 requirement

jraddaoui opened the pull request.

fiver-watson reviewed (approved):
Looks great! 

Lol, so much back on forth on this, but: since I see you've already done the work required to revise the Xenial docs, I supposed we might as well keep them at this point.

fiver-watson reviewed (approved):
Looks great! Push it!

jraddaoui committed:
Remove Xenial docs and update PHP 7.2 requirement

Squashed commit of the following:

commit fbb7be88b4b901f5a4f320f7cb9d78cb4c9ea19d
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Tue Jul 14 20:26:43 2020 +0200

    Re-structure MySQL install docs

commit 4769be51ecfe34b286d9d287bc03b0aaa0a40c3f
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Tue Jul 14 20:06:51 2020 +0200

    Drop Xenial support

commit bb4984f1da597c60360f8f13c7d24d20ab206292
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Mon Jul 13 17:54:22 2020 +0200

    Fix composer links

commit 75667a10d2637626d6f9fabce5f1c465f8a9afd3
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Mon Jul 13 17:48:49 2020 +0200

    Remove references to PHP 7.0

commit 507f93e12199aea2809424d96561f30afacf1527
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Mon Jul 13 17:41:56 2020 +0200

    Update PHP requirement

commit dcc0043d1240eedbf69b655de81995829bbaaf95
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Mon Jul 13 17:40:40 2020 +0200

    Use PHP 7.2 on Xenial

commit c92128f43632c2a5bd39b4282ae5f2c09d74d086
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Mon Jul 13 17:38:57 2020 +0200

    Update install from git section

    - Add composer section to Xenial.
    - Update branches and tarball.
    - Normalice node/npm install and build.

commit f18c21ba558674d6104a9f568053876350d87286
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Mon Jul 13 17:34:11 2020 +0200

    Tweak asynchronous jobs section

    Make clear that the worker needs to be enabled and started.

commit d60fedface898fdea810919fcf4b8cfd3e66b5ba
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Mon Jul 13 15:42:10 2020 +0200

    Clarify MySQL 8 install process

commit 2498fe151b468a814c296e938f745f9f70eb64d5
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Mon Jul 13 15:31:24 2020 +0200

    Update MySQL docs in Xenial

    Same as in Bionic.
